ansible-scylla-node/tasks/common.yml

```diff
@@ -42,6 +42,25 @@
 
 - name: Collect IO settings for the first node of every DC
   block:
+    - name: check for existing io_properties for cloud run
+      stat:
+        path: /etc/scylla.d/io_properties.yaml
+      register: io_prop_cloud_stat
+
+    - name: check for scylla_cloud_io_setup
+      stat:
+        path: /opt/scylladb/scylla-machine-image/scylla_cloud_io_setup
+      register: io_cloud_setup
+
+    - name: Run IO setup in case this is pre-configured environment
+      shell: |
+        /opt/scylladb/scylla-machine-image/scylla_cloud_io_setup
+      become: true
+# if above is not found leverage /opt/scylladb/python3/bin/python3 /opt/scylladb/scylla-machine-image/lib/scylla_cloud_io_setup.py ?
+# or provide our own role runner for pre 2022.2.1 ?
+      when: (io_cloud_setup.stat.exists) and (io_prop_cloud_stat.stat.exists|bool == False) and (present_scylla_disks.stdout|int > 0)
```

Comment:
Why have you added this `present_scylla_disks.stdout|int > 0` check only for cloud?
If you want to add this check, I think it should probably come in the beginning of this block and the role should fail in case the number of disks is 0

Reply:
so the role should work also for situations where you have EBS disk or custom disk or just have everything on root disk
the tuning for various cloud providers should kick in ONLY when there is a raid, resp. /var/lib/scylla mount point (they check for that and won't proceed, so we cannot run that tuning for unsupported setups and have to continue with fallback to iotune - which is DEFAULT way of handling things in 2022 / 5.1)

hence this check is added, I should retest with scenario where you have EBS mounted - I have a bad feeling recent fixes by @syuu1228 will be confused by the fact that EBS volumes can now look like NVME disks ...
( https://github.com/scylladb/scylla-machine-image/blob/master/common/scylla_cloud_io_setup#L238 )
in such case you would get out of box a tuning for local nvme for networked EBS ... 

thanks for pointing this out @igorribeiroduarte , this might be a regression in comparison to new GP3 EBS mounts in AWS which in devtree of kernel look like nvmes !